build.gradle

```diff
@@ -3,6 +3,10 @@ import com.android.builder.core.BuilderConstants;
 buildscript {
   repositories {
     mavenCentral()
+    maven {
+      url 'https://maven.google.com/'
+      name 'Google'
+    }
```

Comment:
Can't these just be `google()`?

Reply:
I believe that would require updating the gradle-wrapper.properties file, `distributionUrl=https\://services.gradle.org/distributions/gradle-2.10-all.zip` up to at least 4.1.  I was just trying to keep the change set as small as possible. The `name 'Google'` attribute can be removed.  I, admittedly, just let Android Studio auto generate the required bits here.

Reply:
Oh wow, I didn't even realize this project was still on such and old version of Gradle. Makes sense then.